__tests__/main.test.js

```diff
@@ -1,25 +1,57 @@
-import Car from '../src/main';
+import { isInputValid } from '../src/utils';
 
-describe('자동차 클래스 테스트', () => {
-  let car = null;
+describe('콘솔 입력에 대한 유효성 검사 동작 테스트', () => {
+  test('사용자 입력란에 "아반떼"을 입력하면 유효성 검사는 통과이다.', () => {
```

Comment:
이런 부분도 context를 조금 더 넣어주면 어떤 걸 테스트 하려는지 의도를 더 쉽게 파악할 수 있을거 같아요!

```suggestion
describe('콘솔 입력에 대한 유효성 검사 동작 테스트', () => {
  describe("5글자 이하의 문자열만 이름으로 사용할 수 있어야 한다.", () => {
      test('"아반떼"을 입력하면 올바른 이름으로 판별한다.', () => {
  })
```